test/ui/test.js

```diff
@@ -26,6 +26,7 @@ const resemble = require('node-resemble');
 var driver = null;
 
 const suiteTimeout = 60000;
+const initTimeout = 10000;
```

Comment:
I assume these units are milliseconds, but it might be nice to make that explicit, either in code or a comment.